src/osw/model/static.py

```diff
@@ -68,9 +139,160 @@ def test_if_empty_list_or_none(obj) -> bool:
             k: v for k, v in self.dict().items() if not test_if_empty_list_or_none(v)
         }
         combined_args = {**self_args, **kwargs}
-        del combined_args["type"]
+        if "type" in combined_args:
+            del combined_args["type"]
         return cls(**combined_args)
 
+    def get_uuid(self) -> Union[str, UUID, NoneType]:
+        """Getter for the attribute 'uuid' of the entity
+
+        Returns
+        -------
+            The uuid as a string or None if the uuid could not be determined
+        """
+        return getattr(self, "uuid", None)
+
+    def get_osw_id(self) -> Union[str, NoneType]:
+        """Determines the OSW-ID based on the entity's uuid.
+
+        Returns
+        -------
+            The OSW-ID as a string or None if the OSW-ID could not be determined
+        """
+        return get_osw_id(self)
+
+    def get_namespace(self) -> Union[str, NoneType]:
+        """Determines the wiki namespace based on the entity's type/class
+
+        Returns
+        -------
+            The namespace as a string or None if the namespace could not be determined
+        """
+        return get_namespace(self)
+
+    def get_title(self) -> Union[str, NoneType]:
+        """Determines the wiki page title based on the entity's data
+
+        Returns
+        -------
+            The title as a string or None if the title could not be determined
+        """
+        return get_title(self)
+
+    def get_iri(self) -> Union[str, NoneType]:
+        """Determines the IRI / wiki full title (namespace:title) based on the entity's
+        data
+
+        Returns
+        -------
+            The full title as a string or None if the title could not be determined.
+        """
+        return get_full_title(self)
+
+
```

Comment:
Functionality hat to be moved back to functions from methods to be applicable to Python classes as in one of the tests